slices/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import "fmt"
+
+func main() {
+
+	// Slices are typed ONLY by the elements they contain.
```

Comment:
How would you create a single slice that could contain any type?
In other words, whats `???` below, so that the code compiles?

```go
s := ???
s = append(s, "foo")
s = append(s, 123)
s = append(s, 'a')
_ = s // stop the compiler from complaining that s is unused
```

Reply:
s := []byte ?

Reply:
Did you try it? (I don't think that's right)

<details>
<summary>Answer</summary>

`s := []any{}` or `s := []interface{}{}` (`any` is a shorthand for `interface{}`)
</details>

Reply:
I should've. Let me 🙇